src/nvim/macros.h

```diff
@@ -127,12 +127,19 @@
     (wp)->w_p_crb = false; \
   } while (0)
 
+
+// Ignore "modulo by 1 operation is meaningless" warning from PVS.
+// The warning is triggered by the second division when an arr is a char array
+// since sizeof(char) is 1. A modulo by 1 operation may normally indicate a logic
+// error, but in this case it's intended.
+
 /// Calculate the length of a C array
 ///
 /// This should be called with a real array. Calling this with a pointer is an
 /// error. A mechanism to detect many (though not all) of those errors at
 /// compile time is implemented. It works by the second division producing
 /// a division by zero in those cases (-Wdiv-by-zero in GCC).
```

Comment:
Update this paragraph instead of adding a new comment block just for the PVS warning.
It's a false positive but it's not stated here.

Reply:
I opted to remove the explanation. I compared the other macro warning suppression and they didn't specify the reason so I thought this would be more consistent.